fipy/solvers/__init__.py

```diff
@@ -127,8 +133,9 @@ def __init__(self):
             raise  SerialSolverError('pyamgx')
         from fipy.solvers.pyamgx import *
         __all__.extend(pyamgx.__all__)
-        from fipy.matrices.scipyMatrix import _ScipyMeshMatrix
-        _MeshMatrix = _ScipyMeshMatrix
+        from fipy.matrices.scipyMatrix import _ScipyRowMeshMatrix as _RowMeshMatrix
+        from fipy.matrices.scipyMatrix import _ScipyColMeshMatrix as _ColMeshMatrix
+        from fipy.matrices.scipyMatrix import _ScipyMeshMatrix as _MeshMatrix
```

Comment:
Could these imports be done more programmatically using importlib? Might be easier to maintain in the long run.

Reply:
Too much for me to wrap my head around right now (#722)

Reply:
> Could these imports be done more programmatically using importlib? Might be easier to maintain in the long run.

OK @wd15, I took a stab at this.